### DISCUSSION TITLE
you can't cmd => winget upgrade with .MSI

### DISCUSSION THREAD

**Discussion by hueldoeu** [rustdesk/rustdesk#8802] [General]
### Bug Description

you can't cmd => winget upgrade with .MSI but you can cmd => winget upgrade with rustdesk-installer.EXE

### How to Reproduce

install rustdesk with rustdesk-installer.MSI and you can't upgrade it with cmd => winget upgrade

### Expected Behavior

install rustdesk with rustdesk-installer.MSI and you can upgrade it with cmd => winget upgrade

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

![image](https://github.com/user-attachments/assets/914b7099-558f-454f-a10d-78407c7fd33c)
if you see this, then you have installed the .MSI

### Additional Context

i have removed the .MSI and live with the installed .EXE

**Comment by hueldoeu** [comment]
probably because only the .EXE is in the winget database.

**Comment by hueldoeu** [comment]
![rusk, what do these settings mean, does it have something to do with scaling Screenshot 2024-07-31 072631](https://github.com/user-attachments/assets/481c325d-3ae9-4630-b20d-16c8c1bad697)
what do these settings mean, does it have something to do with scaling?

**Reply by fufesou** [reply]
What do you mean by scaling? Is it in the following picture?

![image](https://github.com/user-attachments/assets/661fedbf-7225-458d-97b9-25f812af165d)

The options in your screenshot are just default options, most of they can be override in the remote page toolbar.

**Reply by hueldoeu** [reply]
forget scaling, i was dizzy of so much settings. take a look at "dem fokus des entfernten festers folgen" or in english "follow remote window focus"? 


what does that mean? is there a doku where every setting and option is explained perfectly?

**Reply by fufesou** [reply]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#follow-remote-cursor
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#follow-remote-window

https://github.com/rustdesk/rustdesk/pull/7717
